src/common/application-update/progress-of-update-download/progress-of-update-download.injectable.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import createSyncBoxInjectable from "../../sync-box/create-sync-box.injectable";
+import { syncBoxInjectionToken } from "../../sync-box/sync-box-injection-token";
+
+const progressOfUpdateDownloadInjectable = getInjectable({
```

Comment:
It's own injectable because the value needs to be change (and be read) for separate reasons. When progress of download changes, we shouldn't be required to update anything else than the progress. Leads to cleaner code so that you don't have to keep track of the existing values (or create any kind of merge-methods etc.)

Reply:
Resolved for later possible conversation elsewhere.